compositionalityarticle.cls

```diff
@@ -1282,10 +1282,13 @@ class for Compositionality - the open journal for the mathematics of composition
 	\iftoggle{@accepted}
 		{%
 			\newsavebox\@compositionalityacceptedbox
-			
+
 
 			  \newcommand {\logo} {\raisebox{-0.15cm}[0.1cm]{\includegraphics[width=0.035\textwidth, height=0.035\textwidth]{logo.eps}}}
-			\savebox\@compositionalityacceptedbox{{\sffamily\scriptsize \logo\textcolor{compositionalitypink}{Accepted in Compositionality on \ifcsdef{@accepteddate}{\@accepteddate}{}.  Click on the title to verify.}}}}
```

Comment:
The change is self-explanatory: as soon as I am free I will make "DOI" an hyperlink built from the issue/volume number.